access/api.go

```diff
@@ -0,0 +1,136 @@
+package access
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/safing/portbase/api"
+	"github.com/safing/portbase/database/record"
+	"github.com/safing/spn/access/account"
+)
+
+func registerAPIEndpoints() error {
+	if err := api.RegisterEndpoint(api.Endpoint{
+		Path:        `spn/account/login`,
+		Read:        api.PermitAdmin,
+		BelongsTo:   module,
+		HandlerFunc: handleLogin,
+		Name:        "SPN Login",
+		Description: "Log into your SPN account.",
+	}); err != nil {
+		return err
+	}
+
+	if err := api.RegisterEndpoint(api.Endpoint{
+		Path:        `spn/account/logout`,
+		Read:        api.PermitAdmin,
+		BelongsTo:   module,
+		ActionFunc:  handleLogout,
+		Name:        "SPN Logout",
+		Description: "Logout from your SPN account.",
+		Parameters: []api.Parameter{
+			{
+				Method:      "GET",
+				Field:       "purge",
```

Comment:
hm, I guess a `DELETE` HTTP verb would make more sense here? Or at least use POST for that that. `GET` is a dangerous verb for something that performs an action and is not idempotent